build.bash

```diff
@@ -15,7 +15,7 @@ case $1 in
         RUSTFLAGS=--cfg=web_sys_unstable_apis cargo build --target wasm32-unknown-unknown $BUILD_FLAGS --bin $@
         mkdir -p target/generated/
         rm -rf target/generated/*
-        cp -r examples/$1/resources target/generated/ || true
+        cp -r examples/resources target/generated/ || true
```

Comment:
This is intentional - some examples (notably scene-viewer) do have resources that need to be copied. The `|| true` lets this command fail, only copying the resources if they exist.

Reply:
I undid that change, but now there're two commits and i have no idea about squashing and merging them unfortunately.

Reply:
Github can squash for you, no worries.

Reply:
Oh, cool! I had a look but couldn't find anything.

Reply:
Just tried again manually and fucked it up again; feel free to deny/cancel this pull request, i've no idea what i'm doing.